src/mixins/DataReference.js

```diff
@@ -0,0 +1,85 @@
+/**
+ * Gets the screen parent or null if don't have
+ * @returns {object|null}
+ */
+function findScreenOwner(control) {
```

Comment:
So, I feel like this file is the same one I've been seeing across the PRs from this week. Is there any way we can have one single file to serve multiple scenarios? To avoid duplication?

Reply:
In agreement, @caleeli is going to solve this issue in another ticket.